.github/workflows/bandit.yml

```diff
@@ -8,26 +8,32 @@ jobs:
 
       - name: Run checks
         run: |
-          URL="https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files"
-          PR_FILES=$(curl -s -X GET -G $URL | jq -r '.[] | select(.status != "removed") | .filename')
-          for files in $PR_FILES; do
-            extension="${files##*.}"
-            if [[ $extension == 'py' ]]; then
-              changed_files_bandit+=" ${files}"
-            fi
+          PER_PAGE=100
```

Comment:
@dvkruchinin , will https://github.com/marketplace/actions/get-all-changed-files work?

Reply:
I will check. Thank you.

Reply:
@nmanovic 
Yes. It looks more interesting. But renamed files are not taken into account here. Do I need to include them in the check?

Reply:
Files with the ```renamed``` status also need to be checked. Since if the file is changed and renamed, then in the API it has the status ```renamed```. And in the current situation, it will not be checked. I will finalize the PR.

Reply:
@dvkruchinin , if a file is only renamed, it is fine to skip it. If it is renamed and modified and only in `renamed` group, need to address the case.

Reply:
Unfortunately there is no ```renamed and modified``` status.

Reply:
Added ```renamed``` files to check.